document/js-api/index.bs

```diff
@@ -174,7 +176,16 @@ emu-const {
 }
 </style>
 
-This API provides a way to access WebAssembly [[WEBASSEMBLY]] through a bridge to explicitly construct modules from JavaScript [[ECMASCRIPT]].
+<h2 id="intro">Introduction</h2>
+
+By design, the [=scope=] of the WebAssembly [[WEBASSEMBLY]] core specification does not include a description of how WebAssembly programs interact
```

Comment:
Nit: it doesn't look like we have hard wrapping in this file; maybe put a line break after each sentence?